lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -75,7 +77,9 @@
     this.classpath = classpath;
   }
 
-  private void addYarnDriverConfiguration(final File driverFolder, final String jobId, final String jobSubmissionFolder)
+  private Configuration addYarnDriverConfiguration(final File driverFolder,
```

Comment:
Is there a specific reason why the bridge does not use the same submission route as REEFImplementation?